DSA-Campaign-Uplift-Estimation/src/main/webapp/Home/script-home.js

```diff
@@ -10,4 +10,143 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+// Sets the page to always display the first two elements when loading the page.
+var secondChart = 2;
```

Comment:
What if we want to display more than two charts?

Reply:
We agreed that displaying two charts at the same time was better than having the user scroll through the page. Instead the user just clicks next to view the next 2.